include/SFML/Config.hpp

```diff
@@ -178,25 +180,20 @@ namespace sf
     // we can use them without doing any kind of check
 
     // 8 bits integer types
-    typedef signed   char Int8;
-    typedef unsigned char Uint8;
+    using Int8 = std::int_least8_t;
+    using Uint8 = std::uint_least8_t;
```

Comment:
These should not be least, but exact types.
It matters for things like serialization and networking.

Reply:
In the description, I provided an explanation of why it's impossible to achieve major cross-compatibility and exact bit width. I could do that but I didn't know if that's acceptable for SFML to drop the rare cases. Let me know

Reply:
I'd rather have a compile error because of unsupported `std::int16_t`, than subtle bugs at runtime because a wider type is used. The point of these typedefs is exactly to have a precise size -- when SFML doesn't care about the size, it can use `int`, `short` etc.

Do you know platforms that we support, but which don't have `std::[u]intN_t` types?

Reply:
Good point. Supported platforms will be fine. I removed the definitions per @eXpl0it3r's request.